jobs/JGFS_ATMOS_GEMPAK_PGRB2_SPEC

```diff
@@ -33,25 +31,21 @@ export EXT=""
 ##############################################
 # Define COM directories
 ##############################################
-export COMIN=${COMIN:-$(compath.py ${envir}/${NET}/${gfs_ver})/${RUN}.${PDY}/${cyc}/${COMPONENT}}
-export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${gfs_ver}/${NET}.${PDY})/${cyc}/${COMPONENT}/gempak}
-
-export SENDDBN=${SENDDBN:-NO}
-export DBNROOT=${DBNROOT:-${UTILROOT}/fakedbn}
-
-mkdir -m 775 -p ${COMOUT}
-
+export COMIN="${COMIN:-$(compath.py "${envir}"/"${NET}"/"${gfs_ver}")/${RUN}.${PDY}/${cyc}/${COMPONENT}}"
+export COMOUT="${COMOUT:-$(compath.py -o "${NET}"/"${gfs_ver}"/"${NET}"."${PDY}")/${cyc}/${COMPONENT}/gempak}"
```

Comment:
This is not introduced in this PR, but IIRC, we are not supposed to use `compath.py` for locating data under the current application.  Use of `compath.py` should be to find data in upstream applications.
@WalterKolczynski-NOAA Do we have an issue for this?

Reply:
I can answer that...we have https://github.com/NOAA-EMC/global-workflow/issues/289 to address this.

> Use of compath.py should be to find data in upstream applications.

Correct. Upstream or outside applications/models.

Reply:
@KateFriedman-NOAA @aerorahul Is this an oversight on my end or something that should be addressed in another issue/PR?

Reply:
another PR.

Reply:
This also won't work anyway, as it still uses pre-COM-refactor paths.

Reply:
After this PR, I'll update this job for the COM refactor.